testing/citest/tests/openstack_smoke_test.py

```diff
@@ -23,9 +27,20 @@
 endpoint (gate) to determine the managed project it should verify, and to
 determine the spinnaker account name to use when sending it commands.
 
+<<<<<<< HEAD
+Note:
+    This test needs certain environment variables defined in order for the
+    OpenStack client to work. Please refer testing/citest/README.md for more 
+    details.
+
+Sample Usage:
+    PYTHONPATH=testing/citest \
```

Comment:
I guess. Typically you dont run on the spinnaker host. This is assuming that. But it's simpler way to express usage so ok.